src/blockchain/calls/instant.tsx

```diff
@@ -285,10 +286,17 @@ export const offers: CallDef<OffersData, any> = {
 const nullAddress = '0x0000000000000000000000000000000000000000';
 export function proxyAddress$(
   context: NetworkConfig,
-  account: string
+  account: string,
+  proxyRegistryAddress?: string
 ): Observable<string | undefined> {
-  return bindNodeCallback(context.instantProxyRegistry.contract.proxies)(account).pipe(
+  return bindNodeCallback(
+    (proxyRegistryAddress ?
```

Comment:
```javascript
(
 proxyRegistryAddress
    ? web3.eth.contract(proxyRegistry as any).at(proxyRegistryAddress)
    :  context.instantProxyRegistry.contract
).proxies
```

Reply:
What is your concern about it? You mean that for kovan **instantProxyRegistry** and **taxProxyRegistries** are the same? That's right, however for mainnet they will be different (there was different ProxyRegistry on June 2018).

Reply:
My concern is the poorly formatted code :)

Reply:
got it. Fixed.